01-BioDIGS_project_overview.Rmd

```diff
@@ -0,0 +1,66 @@
+
+# (PART\*) BioDIGS Overview {-}
+
+# Background
+
+One critical aspect of an undergraduate STEM education is hands-on research. Undergraduate research experiences enhance what students learn in the classroom as well as increase a student's interest in pursuing STEM careers ^1. It can also lead to improved scientific reasoning and increased academic performance overall ^2. However, many students at underresourced institutions like community colleges, Historically Black Colleges and Universities (HBCUs), tribal colleges and universities, and Hispanic-serving institutions have limited access to research opportunities compared to their cohorts at larger four-year colleges and R1 institutions. These students are also more likely to belong to groups that are already under-represented in STEM disciplines, particularly genomics and data science ^3 ^4.
```

Comment:
Citations should be like `[@ref]` and should be flushed out in the accompanying `book.bib` file. See recent example here: https://github.com/fhdsl/AI_for_Decision_Makers/blob/main/01c-AI_Possibilities-how_ai_works.Rmd

Reply:
Bah. I forgot this isn't just a .md file. Thanks!